### PR TITLE
Added HomeWork 4

### PR DESCRIPTION
**Pull request by oksanaklub**
@MarinaZelenska Added HW4. Created folder with 6th python  files inside. Each file is a single task.
Please review.

### CODE REVIEW

File: HomeWork4/2nd.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+first_name_input = input("Type you name: ")
+reversed_first_name_input = first_name_input[::-1]
+
+if first_name_input == reversed_first_name_input:
+    print(f"String {first_name_input} is Palindrome ")
+else:
+    print(f"String {first_name_input} is not Palindrome ")
```
Comment (MarinaZelenska) [inline HomeWork4/2nd.py:7]: No newline at end of file

File: HomeWork4/1st.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+if operation == '+':
+    print(addition)
+if operation == '-':
+    print(subtraction)
+if operation == '*':
+    print(multiplication)
+if operation == '/':
+    print(division)
+
```
Comment (MarinaZelenska) [inline HomeWork4/1st.py:22]: if the user enters the operation not as you expect?

File: HomeWork4/4th.py [hunk 3]
```diff
@@ -0,0 +1,4 @@
+random_list = ['Tst', 'aBc', 'TEST', 'Hello', 'neW']
+everything_in_upper_case = ' '.join(random_list).upper()
+transfered_string_to_list = everything_in_upper_case.split()
+print(f'Here is example of the sorted list {sorted(transfered_string_to_list)}')
```
Comment (MarinaZelenska) [inline HomeWork4/4th.py:4]: Better case:
print(f'Here is example of the sorted list {sorted(random_list, key=str.upper)}

File: HomeWork4/5th.py [hunk 4]
```diff
@@ -0,0 +1,6 @@
+word_inside_list = [1, 2, 3, 4, 5, 6, [1, 2, 3, ['Win']]]
+only_word = word_inside_list[6][3][0]
+print(f'Take out the word {only_word} from list')
+
+word_again_list = only_word.split()
+print(f'Taking the word "{only_word}" into the list {word_again_list}')
```
Comment (MarinaZelenska) [inline HomeWork4/5th.py:6]: Better case:
print(f'Taking the word "{only_word}" into the list {[only_word]}')

**Review by MarinaZelenska — APPROVED** [review]

File: HomeWork4/2nd.py (hunk 1 above)
Comment (oksanaklub) [reply HomeWork4/2nd.py:7]: fixed

File: HomeWork4/5th.py (hunk 4 above)
Comment (oksanaklub) [reply HomeWork4/5th.py:6]: fixed

File: HomeWork4/1st.py (hunk 2 above)
Comment (oksanaklub) [reply HomeWork4/1st.py:22]: fixed

**Pull request merged**